src/pages/variants/VariantTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FunctionComponent } from 'react';
+import { Table } from 'antd';
+import style from './VariantTable.module.scss';
+import { Spinner } from '../../uikit/Spinner';
+
+type OwnProps = {
+  isLoading: boolean;
+  dataSource: any[];
+};
+
+const columns = [
```

Comment:
if index and key are always the same, I would use a loop to add them dynamically, one typo and you'll have a bug hard to fix